README.md

```diff
@@ -1,90 +1,130 @@
+* Miners receive Flashbots bundles through MEV-Geth. 
+  * there can only be a single bundle per block
+  * the Flashbots bundle will always be at the top slot of the block by design of the system.
+* MEV-Geth picks which bundle to choose for each block.
+  * MEV-Geth will pick the most profitable bundle for the miner.
+  * MEV-Geth will compare the block that includes the bundle vs a vanilla Geth block, and will default back to the vanilla Geth block in case it is more profitable.
+* Only when the trader's bundle is included in a block do they pay the tip associated to their bundle.
+  * a bundle that isn't included costs the sender nothing (ie. no gas fees are paid for failed transactions)
 
-Miners and/or mining pools can receive MEV revenue, which is expected to be higher than revenue from mining regular blocks, for including the most profitable transaction bundles submitted by Flashbots users.
+This infrastructure allows:
+* traders to bypass the Ethereum mempool and avoid their strategy leaking before it is mined on-chain. (ie. being frontrun by generalized frontrunners)
+* traders to save money from avoiding to pay gas fees for failed transactions.
+* miners to receive additional revenue in the form of the bundle tip, in exchange for including the most profitable bundle in the block they mined.
 
-Flashbots aim to provide a "direct fast lane" for transaction inclusion with an efficient, fair and sustainable MEV auction mechanism, potentially reducing network congestion and lowering transaction fees for regular Ethereum users.
+Interested in learning more?
+<br> [Onboard Flashbots Alpha as a trader](https://github.com/flashbots/pm/blob/main/guides/flashbots-alpha.md)
+<br> [Onboard Flashbots Alpha as a miner](https://github.com/flashbots/pm/blob/main/guides/miner-onboarding.md)
+<br> [Transparency Report Jan 2021](https://medium.com/flashbots/flashbots-transparency-report-january-2021-922514de8b8a)
 
-We are currently in the Alpha phase of infrastructure development and are quickly iterating in public towards a permissionless infrastructure, as set forth in the [mission](https://medium.com/flashbots/frontrunning-the-mev-crisis-40629a613752) of Flashbots' organization. We welcome all ecosystem participants to provide early technical feedback via Github issues and pull requests in the relevant [repos](https://github.com/flashbots), create open-source toolings to improve transparency within Flashbots ecosystem, and participate in ongoing discussions on Flashbots' [Discord](https://discord.gg/3TjWjBerRb)!
+We are quickly iterating in public towards a permissionless and fully private infrastructure. We welcome all ecosystem participants to provide early technical feedback via Github issues and pull requests in the relevant [repos](https://github.com/flashbots), create open-source toolings to improve transparency within Flashbots ecosystem, and participate in ongoing discussions on Flashbots' [Discord](https://discord.gg/3TjWjBerRb)!
```

Comment:
private -> public?

Reply:
We are quickly iterating in public towards a permissionless infrastructure with full transaction privacy.